gematria/model/python/loss_utils.py

```diff
@@ -175,7 +167,7 @@ class LossComputation(tf.experimental.ExtensionType):
   absolute_error_percentiles: tf.Tensor
   absolute_percentage_error_percentiles: tf.Tensor
 
-
+@tf.function
```

Comment:
I'm not sure a `tf.function` annotation here is useful. Tracing for tests I would guess just increases the time they take to run. This will also get covered by the `tf.function` annotation on the training step when actually running the model.